js/apps/filter-page/directives/filters.js

```diff
@@ -4,15 +4,29 @@
 	const filtersDirective = () => {
 		const filterLink = function filterLink($scope) {
 			$scope.$watch('filterData', () => {
-				$scope.filterFamilies = $scope.filterData;
+				$scope.filterFamilies = $scope.filterData ? $scope.filterData.map(addFilterId) : $scope.filterFamilies;
```

Comment:
does this work even though ```addFilterId``` is defined below where it's called?

Reply:
Sure does.

Reply:
I'm not seeing a way to do this that won't require a big rewrite. I'm open to ideas.